src/wallet/rpc/spend.cpp

```diff
@@ -968,6 +988,11 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
                              "are replaceable).\n"},
                     {"estimate_mode", RPCArg::Type::STR, RPCArg::Default{"unset"}, "The fee estimate mode, must be one of (case insensitive):\n"
                              "\"" + FeeModes("\"\n\"") + "\""},
+                    {"outputs", RPCArg::Type::ARR, RPCArg::Default{UniValue::VARR}, "New outputs (key-value pairs) that will replace\n"
```

Comment:
The PR description suggests outputs are added, but here it says they are replaced. And I think that's also what the code is doing.

Reply:
I think the PR description hasn't been updated, the author changed the implementation such that the outputs are now replaced. https://github.com/bitcoin/bitcoin/pull/25344#issuecomment-1180660058